applications_tests/lethe-fluid/cahn_hilliard_periodic_boundaries.output

```diff
@@ -0,0 +1,41 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       64
+   Number of degrees of freedom: 243
+   Volume of triangulation:      4
+   Number of Cahn-Hilliard degrees of freedom: 162
+   Number of active cells:       253
+   Number of degrees of freedom: 858
+   Volume of triangulation:      4
+   Number of Cahn-Hilliard degrees of freedom: 572
+   Number of active cells:       988
+   Number of degrees of freedom: 3168
+   Volume of triangulation:      4
+   Number of Cahn-Hilliard degrees of freedom: 2112
+
+*******************************************************************************
+Transient iteration: 1        Time: 0.0005   Time step: 0.0005   CFL: 0       
+*******************************************************************************
```

Comment:
What does this test verify? I understand you have an output and etc, but there's nothing that you are analyzing here... I'm not sure how you will be able to say that this test works or not.

Reply:
I agree it was unclear. I made a different test, with a bubble that travels right and loops back with periodic boundary conditions. We follow the barycenter and see that it increases along X, then starts back at -1. If you like this I'll add something similar for tracer and heat transfer (VOF is already tested)